includes/course-theme/class-sensei-course-theme-compat.php

```diff
@@ -77,6 +77,14 @@ public function the_course_theme_layout() {
 	 * @return string The wrapper template path.
 	 */
 	public function get_wrapper_template( $template ) {
+		// Fix compatibility issue with Divi builder.
+		if (
+			class_exists( 'ET_GB_Block_Layout' )
+			&& method_exists( 'ET_GB_Block_Layout', 'is_layout_block_preview' )
+			&& ET_GB_Block_Layout::is_layout_block_preview()
+		) {
+			return $template;
+		}
```

Comment:
👍 Should be fine for the editor, we only need to load the template wrapper on the frontend, to render block templates.